qa/tasks/cephfs/mount.py

```diff
@@ -184,13 +184,19 @@ def _setup_netns(self):
             for ns in netns_list:
                 ns_name = ns.split()[0]
                 args = ["sudo", "bash", "-c",
-                        "ip netns exec {0} ip addr".format(ns_name)]
-                p = self.client_remote.run(args=args, stderr=StringIO(),
-                                           stdout=StringIO(), timeout=(5*60))
-                q = re.search("{0}".format(ip), p.stdout.getvalue())
-                if q is not None:
-                    found = True
-                    break
+                        "ip netns exec {0} ipd aaddr".format(ns_name)]
```

Comment:
typo? `aaddr`??

Reply:
It seems the "Secure Shell App" editor's problem, sometime switching between chinese and english languages will add some extra chars I typed.

I will fix it.